core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -38,9 +41,60 @@
 
     private static final double[] PERCENTILES = { 0, 0.5, 0.75, 0.9, 0.95, 0.98, 0.99, 0.999, 1.0 };
 
+    private static volatile DistributionStatisticConfig distributionStatisticConfig =
+            DistributionStatisticConfig.builder()
+                                       .percentiles(PERCENTILES)
+                                       .build();
+
+    /**
+     * Sets {@link DistributionStatisticConfig} to {@link MoreMeters}.
```

Comment:
How about:

```
Sets the {@link DistributionStatisticConfig} to use when the factory methods in {@link MoreMeter} create
a {@link Timer} or a {@link DistributionSummary}.
```

...?

Then the remaining sentences ('Any... will use this config.) would be OK to be removed.